its/pom.xml

```diff
@@ -57,13 +57,11 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
       <version>3.6.1</version>
-      <scope>test</scope>
```

Comment:
the whole project is a test project, so either we put the `test` scope to all dependencies, or to none